File: BackEnd/AgendamentoVacina/Core/Entities/Perfil.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Core.Entities
{
    public class Perfil
    {
        public Perfil()
        {
            Usuarios = new HashSet<Usuario>();
        }

        public int Id { get; set; }
        public string Nome { get; set; }

        public virtual ICollection<Usuario> Usuarios { get; set; }
    }
}

File: FrontEnd/back-office/src/app/models/perfil.ts
export class Perfil {
  id: number = 0;
  nome: string = '';
  selecionado:boolean = false;

  constructor(id:number, nome:string, selecionado:boolean) {
    this.id = id;
    this.nome = nome;
    this.selecionado = selecionado;
  }
}
